2019-05-11 17:53:19.1546|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
The request matched multiple endpoints. Matches: 

Fate.WebApi.Controllers.ValuesController.Get (Fate.WebApi)
Fate.Test.Controllers.ValuesController.Get (Fate.Test)

2019-05-11 17:54:23.9633|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
11111sdsds

2019-05-11 17:59:04.1715|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
The request matched multiple endpoints. Matches: 

Fate.WebApi.Controllers.ValuesController.Get (Fate.WebApi)
Fate.Test.Controllers.ValuesController.Get (Fate.Test)

2019-05-11 18:00:24.9923|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
Exception of type 'System.ArgumentException' was thrown.

2019-05-11 23:15:17.2764|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
The request matched multiple endpoints. Matches: 

Fate.WebApi.Controllers.ValuesController.Get (Fate.WebApi)
Fate.Test.Controllers.ValuesController.Get (Fate.Test)

2019-05-11 23:15:38.4071|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
111111111111111

2019-05-11 23:15:47.9698|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
111111111111111

2019-05-11 23:24:05.9813|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
The request matched multiple endpoints. Matches: 

Fate.WebApi.Controllers.ValuesController.Get (Fate.WebApi)
Fate.Test.Controllers.ValuesController.Get (Fate.Test)

2019-05-11 23:24:22.4390|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
111111111111111

2019-05-11 23:24:38.3775|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
Exception of type 'System.ArgumentException' was thrown.

2019-05-11 23:28:52.0587|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
The request matched multiple endpoints. Matches: 

Fate.WebApi.Controllers.ValuesController.Get (Fate.WebApi)
Fate.Test.Controllers.ValuesController.Get (Fate.Test)

2019-05-11 23:29:09.8148|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
111111111111111

2019-05-11 23:29:57.3723|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
Exception of type 'System.ArgumentException' was thrown.

